filewatcher/filewatcher.go

```diff
@@ -308,6 +305,26 @@ func New(ctx context.Context, cfg Config) (*Result, error) {
 	if err != nil {
 		return nil, err
 	}
+	isDir, err := isDirectory(filepath.Clean(cfg.Path))
```

Comment:
What is the reason for the `Clean`s? Is this necessary, or is it to make errors better, or what?

Reply:
Yeah no real reason other than that. If you feel its extraneous I can take them out